reference-implementation/lib/ReadableStream.webidl

```diff
@@ -2,6 +2,8 @@
 interface ReadableStream {
   constructor(optional object underlyingSource, optional QueuingStrategy strategy = {});
 
+  static ReadableStream from(any asyncIterable);
```

Comment:
Should this be fromAsync to match TC39's [Array.fromAsync(asyncIterable) proposal](https://github.com/tc39/proposal-array-from-async/), and to explicitly show that it supports async iterable unlike Array.from? (But that one returns a promise while this one does not 🤔)

Reply:
It makes sense for `Array.from(syncIterable)` and `Array.fromAsync(asyncIterable)` to be two different methods, since they also return different results (`Array` versus `Promise<Array>`).

I don't think such distinction is necessary from `ReadableStream`. We're never going to need *both* `ReadableStream.from()` and `ReadableStream.fromAsync()`. A `ReadableStream` is *inherently* asynchronous, so we never need to return a `Promise<ReadableStream>`.

Node.js also has [`Readable.from()`](https://nodejs.org/dist/latest-v18.x/docs/api/stream.html#streamreadablefromiterable-options), which also accepts both sync and async iterables.

I think `ReadableStream.from()` is still be the better option. It makes it clear that this is *the* primary method to convert anything into a `ReadableStream`. A `from()` method usually also implies "idempotency", that is: passing an array to `Array.from()` returns a new array. On the other hand, `fromAsync()` feels more "foreign".

Reply:
Makes sense, LGTM then.